TWLight/users/migrations/0067_auto_20201120_2002.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.0.11 on 2020-11-20 20:02
```

Comment:
It would be great to change the name of this file to indicate that fields are being removed from the `Editor` model.

Reply:
Fair. Done!